logstash-core/src/main/java/org/logstash/execution/plugins/ConfigValueConverter.java

```diff
@@ -0,0 +1,5 @@
+package org.logstash.execution.plugins;
+
+public interface ConfigValueConverter<T> {
```

Comment:
@danhermann I don't think we can do something like this :( If we expose this kind of interface, we're adding flexibility here that we can't properly map to UI functionality.
I like the general idea of mapping from a `<String, String>` though. We just have to constrain the possible types more than this interface does :) Let's talk about it offline later.